NodeJS/src/AgentClient.ts

```diff
@@ -9,27 +9,38 @@ export class AgentClient {
 
     private restClient: RestClient
     private subAccountId: string
+    private defaultSkillNumber: number
 
     constructor(subAccountId: string, restClient: RestClient) {
         this.restClient = restClient
         this.subAccountId = subAccountId
+        this.defaultSkillNumber = -1
+    }
+
+    getDefaultSkillNumber(){
+        return this.defaultSkillNumber
     }
     /**
      * Create Agent and Station. Upon success, returns agent object and station object
      * @param agentUsername min length 2, max length 25, must pass ^[-.@\w]+$
      * @param agentPassword min length 8, max length 32, must have a uppercase character, must have at least one lowercase char, no whitespace, must contains a number, must contain one of ~!@?#$%^&*_
-     * @param skillsWithPriority this could retrieved via #getSkillNumbers()
      */
-    public async createAgentAndStation(agentUsername: string, agentPassword: string, skillsWithPriority: SkillPriority[]) {
+    public async createAgentAndStation(agentUsername: string, agentPassword: string) {
         if (!isValidPassword(agentPassword)) {
             return Promise.reject("invalid password")
```

Comment:
do you need to worry about translating this text?

Reply:
no.

Reply:
lucky